### DISCUSSION TITLE
License on book projects

### DISCUSSION THREAD

**Discussion by fuhrmanator** [quarto-dev/quarto-cli#3015] [Q&A] [answered]
I'm trying to activate the https://quarto.org/docs/authoring/appendices.html#license feature on a book project, and I don't see anything in the Appendices, despite having defined `license: "CC BY-SA"` (or whatever). The documentation is a little vague, since the bread crumbs to that feature are in "authoring" which almost seems like "journal articles" rather than books. 

Is this feature working for books? I can of course write the license info by hand in the Preface (which is where I think a lot of CC-* books do it). Anyway, I'd be pleasantly surprised if the text were generated in French (the language of my book, which might be the reason it's not working?).

**Comment by dragonstyle** [comment, marked as answer]
Currently the appendices feature (and the license feature and others) aren't implemented for books, only HTML documents. I do think it would be nice to enable this (perhaps the appendices feature but certainly the license feature), and we're hoping to invest some time in improving our book support in an upcoming release, but for the time being I think it would be best for you to write the license info out yourself.

**Reply by fredguth** [reply]
If you plan to improve book support, please create a discussion topic where we could add our opinion. I do have many suggestions.

**Reply by fuhrmanator** [reply]
I echo @fredguth 's sentiment - several things can be improved and I would be happy to give feedback on my experience.
